app/src/main/res/layout/administrator_controls_download_permissions_view.xml

```diff
@@ -18,9 +18,9 @@
       style="@style/CaptionViewStart"
       android:layout_width="match_parent"
       android:background="@color/mid_grey_30"
-      android:paddingStart="16dp"
+      android:paddingStart="@dimen/administrator_controls_download_permissions_view_upper_text_view_padding_start"
       android:paddingTop="28dp"
-      android:paddingEnd="16dp"
+      android:paddingEnd="@dimen/administrator_controls_download_permissions_view_upper_text_view_padding_end"
```

Comment:
Ditto. Remove `upper` and use `administrator_controls_download_permissions_view_download_permissions_text_view_padding_end` instead